config/local-dev.yml

```diff
@@ -11,14 +11,14 @@ analysis:
       chaincontext: b11b369e0da5bb230b220127f5e7b242d385ef8c6f54906243f30af63c815535
       from: 2550000
   storage:
-    endpoint: postgresql://indexer:password@localhost:5432/indexer?sslmode=disable
+    endpoint: postgresql://rwuser:password@localhost:5432/indexer?sslmode=disable
```

Comment:
Should `config/docker-dev.yml` be changed too?

Reply:
On second thought, we should just merge those two files.

Reply:
All good. Lemme just do a separate small PR for those.

Reply:
Okay, can improve `docker-dev.yml` https://github.com/oasisprotocol/oasis-indexer/pull/155, but can't de-dup the files because `migrations: file:///storage/migrations` is an absolute path due to Docker usage.